supporting_files/AtlasSiteConfig.yml.default

```diff
@@ -87,11 +87,11 @@ atlas_curated_accessions_file: atlas_curated_accessions.yml
 # File containing accessions of experiments that don't have R experiment summaries.
 no_r_data_accessions_file: no_r_object
 
-# URL to get ADF info from ArrayExpress.
-arrayexpress_adf_info_url: http://peach.ebi.ac.uk:8480/api/array.txt?acc=
+# URL to get ADF info from ArrayExpress.  Replace URL because PEACH has been deprecated.
+arrayexpress_adf_info_url: http://codon-plantain-01:8480/arrays?acc=
 
-# URL to get privacy info from ArrayExpress
-arrayexpress_privacy_info_url: http://peach.ebi.ac.uk:8480/api/privacy.txt?acc=
+# URL to get privacy info from ArrayExpress.
+arrayexpress_privacy_info_url: https://www.ebi.ac.uk/biostudies/api/v1/search?type=study&accession=
```

Comment:
As you know, this new query is going to return the result in JSON format. 

It would be great to double-check that where `arrayexpress_privacy_info_url` is used, it is fine with getting the result in json

Reply:
The `arrayexpress_privacy_info_url` and `arrayexpress_adf_info_url` variables don't seem to be used in any of our Gitlab and Github repos (other than here, where they are defined).  Perhaps these are used in files that are not in a repo (or replaced and are no longer in use).  I checked with the curators, but they also don't know where these variables are used.